src/lib/mail/sendEmail.server.ts

```diff
@@ -72,19 +73,40 @@ export const sendTransactionalEmail = async ({
 export const sendEmailNotification = async ({
   name,
   email,
-  message
+  message,
+  type
 }: {
   name: string;
   email: string;
   message: string;
+  type: FormType;
 }) => {
   name = escapeHTML(name);
   email = escapeHTML(email);
   message = escapeHTML(message);
 
+  const allEnvsEmail = [
+    {
+      env: env.NOTIFICATION_EMAIL_ADDRESS_GET_A_QUOTE,
+      type: 'quote'
+    },
+    {
+      env: env.NOTIFICATION_EMAIL_ADDRESS_APPLY_TO_POSITION,
+      type: 'career'
+    },
+    {
+      env: env.NOTIFICATION_EMAIL_ADDRESS_TALK_TO_US,
+      type: 'contact'
+    }
+  ] satisfies { env: string; type: FormType }[];
```

Comment:
I would rather have a `formTypeToDestinationEmail`
This way we would avoid:
- having a value with `env` that is not really an env and is the value of the env not the env name
- needing to find the value over an array. It would be direct with the access operator `[]`